locales/it/index.yml

```
@@ -250,6 +250,10 @@ authentication:
     paragraph3: Proseguendo dichiari di accettare le
     tosLinkText: regole sul trattamento dei dati personali
     continue: Continua
+  cie:
+    temporarilyBlockedCie: !include cie/temporary_blocked_cie.md
+    temporarilyBlockedCieDoneButton: "Fatto, carta sbloccata!"
+    temporarilyBlockedCieOpenCieID: "Apri CieID"
```

```suggestion
    temporarilyBlockedCieOpenCieID: "Scarica / Apri CieID"
```
https://projects.invisionapp.com/share/QRUHJOUWSYV#/screens